src/main/java/redis/clients/jedis/Protocol.java

```diff
@@ -133,7 +137,7 @@ private static String[] parseTargetHostAndSlot(String clusterRedirectResponse) {
 
   private static Object process(final RedisInputStream is) {
     final byte b = is.readByte();
-    //System.out.println((char) b);
+    //System.out.println("BYTE: " + (char) b);
```

Comment:
remove?

Reply:
as we are still in development, these things help me to debug